From 2df73cb4050bf11d5f8d4583cbaa11648619549d Mon Sep 17 00:00:00 2001
From: Christian Kastner <debian@kvr.at>
Date: Mon, 31 Mar 2014 18:19:49 +0200
Subject: [PATCH] Remove redundant Section in binary package of library

The section "libs" is identical to the section of the source package and is
therefore redundant. This patch removes it, thereby also silencing a lintian
info for binary-control-field-duplicates-source field "section".
---
 lib/debianl/control | 1 -
 1 file changed, 1 deletion(-)

diff --git a/lib/debianl/control b/lib/debianl/control
index d7db427..a559205 100644
--- a/lib/debianl/control
+++ b/lib/debianl/control
@@ -16,7 +16,6 @@ Description: <insert up to 60 chars description>
  <insert long description, indented with spaces>
 
 Package: #PACKAGE#BROKEN
-Section: libs
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}
 Description: <insert up to 60 chars description>
-- 
1.8.5.3

